code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -221,6 +221,20 @@ var/list/ai_verbs_default = list(
 		setup_icon()
 	eyeobj.possess(src)
 
+	//Link the existing borgs to the AI
+	for (var/mob/living/silicon/robot/R in silicon_mob_list)
+		//If its not slaved we dont link it
+		if(!R.is_slaved())
```

Comment:
`is_slaved()` will always return false when there's no AI at roundstart, because connected_ai will be null since there wasn't an AI available when the borg initialized